src/server/seeds/development/03_users.js

```diff
@@ -0,0 +1,80 @@
+exports.seed = function (knex) {
+  return knex('users')
+    .del()
+    .then(function () {
+      return knex('users').insert([
+        {
+          full_name: 'Nick Wahlberg',
+          email: 'wahl123@nick.com',
+          address: 'Enghavevej 21',
+          zipcode: '1234',
+          city: 'Copenhagen',
+          country: 'Denmark',
+        },
```

Comment:
In all users you have missed **created_at** .

Reply:
Hi @zmzade , actually its the default value, so i think its not necessary to give the timestamp individually for all the users, we can give explicitely but i just left it as a default date time value. Anyway thanks for your review 👍

Reply:
I tested on the local server and it seems okay. :)

Reply:
correct